src/readpst.c

```diff
@@ -1151,6 +1152,31 @@ void write_embedded_message(FILE* f_output, pst_item_attach* attach, char *bound
     DEBUG_RET();
 }
 
+/**
+ * Backslash-escape quotes in the given string.
+ */
+char *escape_quotes(char *inp) {
```

Comment:
Yep. (I guess this is a bug in the spec? https://stackoverflow.com/questions/7886782/what-is-the-exact-syntax-and-semantics-of-a-quoted-string-in-the-http1-1-rfc2616 but I'd have to check for sure if this is actually the definition we're using here...).